packages/lib/src/interactions/SelectToZoom.tsx

```diff
@@ -22,14 +23,15 @@ function SelectToZoom(props: Props) {
   const { width, height } = useThree((state) => state.size);
   const camera = useThree((state) => state.camera);
 
-  const { xVisibleDomain, yVisibleDomain } = useVisibleDomains();
+  const { xVisibleDomain, yVisibleDomain } = useCameraState(
+    getVisibleDomains,
+    []
```

Comment:
If I default the `deps` parameter to `[]` in `useCameraState`, then the linting rule `exhaustive-deps` no longer warns you if you don't pass a dependency array even though you need one.

Reply:
It's the same for `useCallback`, right? So it's inline with react.